src/pages/nerd-bytes.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import cx from 'classnames';
+import { css } from '@emotion/core';
+
+import SEO from '../components/Seo';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import PageLayout from '../components/PageLayout';
+import FeatherIcon from '../components/FeatherIcon';
+import { PageContext } from '../components/PageContext';
+import styles from './index.module.scss';
+import podcastBadge from '../images/podcasts/podcasts-badge.png';
+import Video from '../components/Video';
+
+const NerdbytesPage = () => {
+  return (
+    <PageContext.Provider>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="New Relic Nerd Bytes" />
+
+        <PageLayout.Content>
+          <section
+            css={css`
+              margin-top: 0;
+            `}
+            className={cx(styles.intro, 'intro-text')}
+          >
+            <div className={styles.introText}>
+              <p>
+                <Link
+                  className={styles.externalLink}
+                  to="https://www.youtube.com/playlist?list=PLmhYj7Jl81JEV-llIDkCVC05tD7fbOv_b"
+                >
+                  Nerd Bytes is a video podcast series
+                  <FeatherIcon
+                    className={styles.externalLinkIcon}
+                    name="external-link"
+                  />
+                </Link>{' '}
+                produced by engineers and practitioners _for_ engineers and
```

Comment:
Just checking this appears as italicised yes? no need for <em> tag? Not sure how to see the preview